README.md

```diff
@@ -1,2 +1,58 @@
-# chmln
-Easily add Chameleon to your product
+# Chameleon JavaScript Library
+
+Easily add Chameleon to your product!
+
+The Chameleon JavaScript Library is a set of methods attached to a global `chmln` object intended to be used by websites wishing to display Chameleon Experiences to users.
+
+## Installation via NPM
+
+This library is available as a package on NPM. to install into a project using NPM with a front-end packager such as Browserify or Webpack:
+
+```
+  npm install --save @chamaeleonidae/chmln
+```
+
+You can then require the lib like a standard Node.js module:
+
+```javascript
+const chmln = require('@chamaeleonidae/chmln');
+
+chmln.init('YOUR_TOKEN');
+```
+
+## Usage
+
+First, follow the installation process above, the script loads asynchronously and won’t affect your page loading times.
```

Comment:
I would add a full stop instead of comma between the two sentences:
First, follow the installation process above **.** The script loads asynchronously and won’t affect your page loading times.